### ISSUE TITLE
prefer inline, notes

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1180]
first use, not ok if

- 	CLEAR
- 	APPEND
- 	CREATE OBJECT TYPE, or anything with Generic?
- 	do not report for FORM parameters

ok in

- 	MOVE, if not used in the Source part(right side)
- 	CATCH INTO
- 	LOOP AT INTO/ASSIGNING
- 	READ TABLE INTO/ASSIGNING

```abap
CONCATENATE |foo| |bar| INTO DATA(lv_filename).
WRITE lv_filename.
```

**Comment by FreHu** [comment]
also see #1290
